src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx

```diff
@@ -21,7 +21,12 @@ const VirtualMachinesInstancesRow: React.FC<VirtualMachinesInstancesRowProps> =
   return (
     <>
       <TableData activeColumnIDs={activeColumnIDs} id="name">
-        <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} />
+        <ResourceLink
```

Comment:
there is a blank line here?

Reply:
nope, it's the standard formatting made by eslint